packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -38,20 +37,33 @@ describe('background sync queue test', () => {
     onRetryFail: onRetryFail,
   };
 
-  const backgroundSyncQueue
-    = new workbox.backgroundSync.test.BackgroundSyncQueue({
+  let backgroundSyncQueue;
+
+  beforeEach(async ()=>{
```

Comment:
Please use `function() {}` here. This is because mocha does some weird binding stuff that means fat arrows have unexpected results if you tried to do this.timeout or this.retries.